packages/react/src/file-upload/FileUpload.tsx

```diff
@@ -313,18 +326,19 @@ export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
 											</Text>
 										</span>
 										{maxSize ? (
-											<Text color="muted">
+											<Text color="muted" id={fileSizeDescriptionId}>
 												{multiple ? 'Each file' : 'File'} cannot exceed{' '}
 												{formattedMaxFileSize}.
 											</Text>
 										) : null}
 										{accept ? (
-											<Text color="muted">
+											<Text color="muted" id={acceptedFilesDescriptionId}>
 												Files accepted: {acceptedFilesSummary}.
 											</Text>
 										) : null}
 									</Stack>
 									<Button
+										aria-describedby={buttonAriaDescribedBy}
```

Comment:
It still renders the `aria-decribedby` prop as an empty string. Should probably remove it entirely

Reply:
Should be fixed now 🙌